.github/workflows/image.yaml

```diff
@@ -0,0 +1,180 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on pull requests or merge to main/release branches
+name: image
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+      - release-*
+  push:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        driver:
+          - 470.223.02
+          - 535.161.07
+          - 550.54.14
```

Comment:
We don't have this released yet. Might want to comment this out before merging

Reply:
It is okay to keep. We have not published it to nvcr.io yet, but it is valid to build 550 images in our GH actions.

Reply:
I thought we wait till the official release of the first 550 image, but it does make sense to build pre-release images of R550